templates/github/.github/workflows/scripts/install.sh.j2

```diff
@@ -37,8 +37,6 @@ PULPCORE=./pulpcore
 {%- endif %}
 if [[ "$TEST" == "plugin-from-pypi" ]]; then
   PLUGIN_NAME={{ plugin_name }}
-elif [[ "${RELEASE_WORKFLOW:-false}" == "true" ]]; then
-  PLUGIN_NAME=./{{ plugin_name }}/dist/{{ plugin_snake }}-$PLUGIN_VERSION-py3-none-any.whl
```

Comment:
whl does not consider MANIFEST.in
https://stackoverflow.com/questions/24347450/how-do-you-add-additional-files-to-a-wheel

Reply:
I think we generally don't want additional files to be included in wheels at all. It's designed to just be application code (my understanding). I use the term application code to indicate disinclusion of test python code (which is still python code). The one thing every distribution type (including wheel) needs to include is the license file.